library/src/tests/resources/compare.qs

```diff
@@ -4,49 +4,50 @@ namespace Test {
     open Microsoft.Quantum.Diagnostics;
 
     internal operation CompareWithBigInt(
-        name: String,
+        name : String,
         bitwidth : Int,
         quantumComparator : (BigInt, Qubit[], Qubit) => Unit,
-        classicalComparator : (Int, Int) -> Bool) : Unit {
-        
+        classicalComparator : (Int, Int) -> Bool
+    ) : Unit {
+
         for n in 1..bitwidth {
             use qs = Qubit[n];
             use t = Qubit();
 
-            for a in 0 .. 2^n+1 { // We want b to have more bits sometimes...
-                for b in 0 .. 2^n-1 {
+            for a in 0..2 ^ n + 1 {
```

Comment:
Have you considered not putting spaces around ^ (as it seems to be the highest priority operator)?

Reply:
I think the consensus around this has shifted to not wanting the spaces (I also think it looks better without spaces), so I will add a rule to the formatter to special-case the `^` to not have spaces around it.

Reply:
removed spacing around the `^` operator